docs/src/userguide/language_basics.rst

```diff
@@ -381,6 +381,48 @@ Sometimes Cython will complain unnecessarily, and sometimes it will fail to
 detect a problem that exists. Ultimately, you need to understand the issue and
 be careful what you do.
 
+
+Type Casting
+------------
+
+As C uses the ``"("`` and ``")"`` for casting types, Cython uses ``"<"`` and ``">"``
+to signify a casting. For example::
+
+    cdef char *p
+    cdef float *q
+    p = <char*>q
+
+If one of the types is a Python object for ``<type>x``, Cython will try to do a coercion.
```

Comment:
I do not understand what you are trying to say with this sentence.

Reply:
I think what it means is this:
"When casting a C value to a Python object type or vice versa, Cython will attempt a coercion."

Reply:
Thanks, the reference guide isn't always easy to understand.